frontend/src/actions/common.js

```diff
@@ -64,7 +64,7 @@ export function handleErrors(reason, originalFunc){
 		return (dispatch) => {
 			dispatch({
 				type: NOT_AUTHORIZED,
-				reason: reason.body.detail,
+				reason: reason.body.detail !== undefined? reason.body.detail : "Forbidden",
```

Comment:
could you even try check if `body` is a field of reason by doing `reason.body !== undefined && reason.body.detail !== undefined ? ...`?

Reply:
Makes sense, I'll modify the code. Thanks ~